cloud/ec2_fleet_test.go

```diff
@@ -13,10 +13,15 @@ import (
 	"github.com/evergreen-ci/evergreen/db"
 	"github.com/evergreen-ci/evergreen/model/distro"
 	"github.com/evergreen-ci/evergreen/model/host"
+	"github.com/evergreen-ci/evergreen/testutil"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+func init() {
+	testutil.Setup()
```

Comment:
This doesn't seem related to linting. What is this change for?

Reply:
I added this while troubleshooting tests in this file - turns out the issue was caused by a change to a regex the linter suggested to use raw string (`...`) notation. This breaks things so I suppressed the linter on that line. Anyways yes, removing this line.